File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
import java.util.*;

class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        
        int[] arr = {a,b};
        Arrays.sort(arr);
        
        int max = arr[1];
        int min = arr[0];
        for(int i=min; i<=max; i++){
            answer += i;
        }
        return answer;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.swift
func solution(_ a:Int, _ b:Int) -> Int64 {
    
    let start = min(a, b)
    let end = max(a, b)
    
    let n = end - start + 1
    let sum = n * (start + end) / 2
    return Int64(sum)
}